onnxruntime/core/optimizer/attention_fusion_helper.h

```diff
@@ -877,8 +877,8 @@ bool MatchInputMaskSubgraph(const Graph& graph, const Node& layer_norm, const No
 }
 
 struct MatchPastResult {
-  NodeArg* past;
-  NodeArg* present;
+  NodeArg* past = nullptr;
```

Comment:
Shall we stick with {} for uniformity?

Reply:
Currently we don't have a unified coding style for this.